build.py

```diff
@@ -43,12 +44,24 @@ def _rmtree_error_handler(func, path, exc_info):
     else:
         raise
 
+def _get_stripped_tar_members(tar):
+    for tarinfo in tar.getmembers():
+        path = tarinfo.name.split('/')
+        if len(path) == 1:
+            if tarinfo.isdir():

```

Comment:
I dont think so, the continue is necessary to skip extraction of the top level directory.